product-service-description/DE-business-model.md

```diff
@@ -0,0 +1,37 @@
+# Business Modell von fiskaltrust in Deutschland
+
+## Business Model Canvas
+
+[Download der Schritt-für-Schritt Präsentation des Business Modelles im Powerpoint-Format](media/business-model-market-de.pptx).
+
+![business-model-market-de](media/business-model-market-de.png)
+
+### Valuestream der fiskaltrust.Middleware
+
+fiskaltrust bietet in Deutschland eine für den deutschen Markt angepasste Compliance as a Service Lösung für Kassenhersteller an. 
+
+Diese wird als fiskaltrust.Middleware über den strategischen Partner in Deutschland, der Audicon GmbH, bzw. über die Webseite von fiskaltrust und über Empfehlungen bestehender Kunden vertrieben. 
+
+Die fiskaltrust.Middleware ist kostenlos, um eine möglichst hohe Verbreitung am Markt zu erreichen.
+
+fiskaltrust pflegt direkten Kontakt mit Kassenherstellern, um diese bei der Integration der fiskaltrust.Middleware zu unterstützen. Zu den weiteren Schlüsselaktivitäten zählen der Erwerb und Erhalt von Know-How im Bereich Fiskalisierung sowie deren Gesetzesgrundlagen, und die Entwicklung der benötigten Fiskalisierungsplattform.
+
+Das Fiskalisierungs-Know-How, vertrauensbildende Maßnahmen wie Zertifizierungen und strategische Partnerschaften, sowie das Wissen um die Verwendung der Kassensysteme zur kontinuierlichen Verbesserung des Produktangebotes, zählen zu den wichtigsten Ressourcen von fiskaltrust.
+
+Die größten Ausgaben entstehen durch Developer, Experten sowie strategische Partnerschafte und Zertifizierungen.
+
+### Valuestream der All-in-one-Fiskalisierungslösungen
+
+Zusätzlich zur fiskaltrust.Middleware bietet fiskaltrust noch All-in-one-Fiskalisierungslösungen für Kassenbetreiber an, welche überwiegend auf die von den Kassenherstellern integrierte fiskaltrust.Middleware aufsetzen - wie etwa Addons oder Produktbundles inklusive (Hardware) TSE.
+
+Der Vertrieb erfolgt hierbei über das bestehende Händlernetzwerk des Kassenbetreibers, der bereits die fiskaltrust.Middleware integriert hat.
+
+Zu den wichtigsten Erlösen zählen einmalige-Einnahmen durch den Wiederverkauf der TSE, sowie wiederkehrende jährliche Einnahmen durch Subscriptions (Service-Mietmodell).
+
+Aufgrund des indirekten Vertriebsmodelles über Kassenhändler kommt es in der Regel zu keinem direkten Kontakt zwischen fiskaltrust und Kassenbetreibern.
+
```

Comment:
Vorschlag: "fiskaltrust verkauft die Produkte ausschließlich über Kassenhändler und nicht direkt an die Endkunden (Kassenbetreiber).
Daher bitte wir alle interessierten Endkunden bei produktbezogenen Fragen sowie bei Support-Anfrage, sich an ihren jeweiligen Betreuer ihres Kassenhändlers zu wenden. Dieser kennt auch Ihre individuelle Konfiguration sowie Ihre Systemumgebung. Bei rechtlichen Fragen - z.B. zu der gesetzliche erforderlichen "Verfahrensdokumentation" empfehlen wir, dass Sie sich z.B. mit ihrem/r SteuerberaterIn in Verbindung setzen."